libbeat/scripts/Makefile

```diff
@@ -142,15 +143,18 @@ prepare-tests:
 .PHONY: unit-tests
 unit-tests: ## @testing Runs the unit tests with coverage.  Race is not enabled for unit tests because tests run much slower.
 unit-tests: prepare-tests
+	go test -i ${GOPACKAGES}
```

Comment:
From the help of `go test`:

```
	-i
	    Install packages that are dependencies of the test.
	    Do not run the test.
```

And indeed `go test -i` doesn't actually run the tests. However, in master `make unit-tests` seems to work fine, so I'm not quite sure what's going on. But this makes me think adding `-i` to the test commands might not do what we expect.